salt/modules/google_chat.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+'''
+Module for sending messages to google chat.
+
+.. versionadded:: Fluorine
+
+To use this module you need to configure a webhook in the google chat room
+where you would like the message to be sent, see:
+
+    https://developers.google.com/hangouts/chat/how-tos/webhooks
+'''
+# Import Python Libs
+from __future__ import absolute_import
```

Comment:
Can you add `print_function` and `unicode_literals` to this `__future__` import? We need that for PY2/PY3 compatibility. :)